src/guides/v2.3/frontend-dev-guide/theme-best-practice.md

```diff
@@ -27,66 +27,60 @@ We recommend using the following best practices when developing themes:
 1. Use [the CSS critical path]({{ page.baseurl }}/frontend-dev-guide/css-topics/css-critical-path.html) to render the page much faster.
 1. Always keep the text translatable. To ensure text used within your Magento templates can be translated, wrap it within the translate function:
    Example:
-   ```xml
+   ```php
    <a href="#"><?= __('Click to download'); ?></a>
    ```
 
 1. Make use of the mobile-first approach when inheriting blank or Luma themes.
 1. Magento has a set of [coding standards]({{ page.baseurl }}/coding-standards/bk-coding-standards.html) for both back-end and front-end technologies. Refer to them when needed.
 1. Do not repeat work while styling. Instead, create a class or mixin and call them when needed.
 1. While styling any custom module, add the styling within the module, instead of adding it to the design theme. This way, the style will not be loaded unless the module is called. For example `app/code/Company/Module/view/frontend/web/css/source/_module.less`.
-1. While styling a custom theme, add styles to seperate less files, instead of appending to a single file. This way, styles are easier to track down and debug. For example, here is a `app/design/frontend/Company/ThemeCustom/web/css/style-l.less` with styles for desktop.
+1. While styling a custom theme, add styles to seperate less files, instead of appending to a single file. This way, styles are easier to track down and debug.
 
-   In `style-l.less`, define:
+   For Reference check this file `[Magento_Blank_Theme_Path]/web/css/_styles.less`:
 
    ```less
-   // Define default styles from parent
-   ...
-   // Extra styles
-   @import 'source/_custom-styles.less';
+   @import 'source/lib/_lib.less'; // Global lib
+   @import 'source/_sources.less'; // Theme styles
+   @import 'source/_components.less'; // Components styles (modal/sliding panel)
    ```
 
-   and then in ` _custom-styles.less`:
-
+   Magento already styled and having many ready made component(s),
+   To find already existing component(s) in blank theme: `[Magento_Blank_Theme_Path]/web/css/source/_sources.less` and  `[Magento_Blank_Theme_Path]/web/css/source/_components.less`.
```

Comment:
Hi @kanhaiya5590. Could you check this sentence, please? We have the purpose "to find" but the actionable point "what to do?" seems to be missing.

Reply:
updated